config_defaults/subtests/docker_cli/run_volumes.ini

```diff
@@ -2,6 +2,9 @@
 subsubtests = volumes_rw, volumes_one_source
 #: These must be setup prior to testing
 __example__ = host_paths, cntr_paths
+#: Command(s) required to succeede prior to testing (if any)
+#: e.g. setsebool virt_sandbox_use_nfs
```

Comment:
Missing value (eg 1/true/on).

Given that the comment includes "(s)" for multiple commands, should the comment also specify "semicolon-separated"? It doesn't seem worth the effort to make this a list.